style.css

```diff
@@ -153,63 +160,158 @@ i:hover {
 }
 
 /* Authentication -pages */
-.signup-form, .login-form{
-    padding:1rem;
+.signup-form,
+.login-form {
+    padding: 1rem;
 }
-.signup,.login {
+
+.signup,
+.login {
     margin: auto;
     padding: 1rem;
     border-radius: 0.5rem;
     background-color: #fff;
     box-shadow: var(--all-side-shadow);
 }
-.signup:hover ,.login:hover{
+
+.signup:hover,
+.login:hover {
     box-shadow: var(--all-side-bold-shadow);
 }
 
-.signup .heading, .login .heading {
+.signup .heading,
+.login .heading {
     text-align: center;
     font-family: var(--heading-font);
 }
-#signup .input-block-name{
-    display:flex;
+
+#signup .input-block-name {
+    display: flex;
 }
-#signup .input-block , #login .input-block{
+
+#signup .input-block,
+#login .input-block {
     display: flex;
     flex-direction: column;
     align-items: stretch;
 }
-.input-block-check{
+
+.input-block-check {
     padding: 1rem;
 }
-.input-block-check .btn-link{
+
+.input-block-check .btn-link {
     text-decoration: none;
     font-size: smaller;
 }
+
 .input-block input {
     border-radius: 0.2rem;
     border: 1px solid var(--light-grey);
     padding: 0.5rem;
 }
-.input-block label{
+
+.input-block label {
     margin-bottom: 0.5rem;
 }
-.signup-btn , .login-btn{
+
+.signup-btn,
+.login-btn {
     background-color: var(--text-col);
     color: var(--back-color);
-    padding:0.5rem;
+    padding: 0.5rem;
 }
 
-.signup-to-login,.login-to-signup {
+.signup-to-login,
+.login-to-signup {
     text-decoration: none;
     color: var(--text-col);
     text-align: center;
 }
 
-.signup-to-login .fa ,.login-to-signup .fa{
+.signup-to-login .fa,
+.login-to-signup .fa {
     padding-left: 0.2rem;
 }
 
+/* product-page */
+.product-container {
+    display: grid;
+    grid-template-columns: 200px 1fr;
+    gap: 1rem;
+}
+
+.filter-aside {
+    display: flex;
+    flex-direction: column;
+}
+
+.prod-head {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem;
+}
+
+.prod-head .prod-brand,
+.prod-heading {
+    font-size: x-large;
+    font-weight: bold;
+    text-align: center;
+}
+
+.prod-price,
+.prod-category,
+.prod-rating {
+    margin: 1rem;
+    display: flex;
+    flex-direction: column;
+}
+
+.prod-price input[type="range"i],
+.prod-category input,
+.prod-rating input,
+.prod-rating .fa {
+    accent-color: var(--text-col);
+}
+
+.filter-heading {
+    font-weight: bold;
+    margin-bottom: 0.5rem;
+}
```

Comment:
Lower than 1 rem you can use Pixels.

Reply:
I got a review to use pixels or rem not to mix them